### ISSUE TITLE
Mass Obsoletion: focus on branch 'musculoskeletal system disorder' (MONDO:0002081)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6751]
branch: 'musculoskeletal system disorder' (MONDO:0002081)

Terms to obsolete: 
obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0017240 | acrodysostosis with multiple hormone resistance
http://purl.obolibrary.org/obo/MONDO_0017422 | adactyly of hand
http://purl.obolibrary.org/obo/MONDO_0018239 | aggrecan-related bone disorder
http://purl.obolibrary.org/obo/MONDO_0020343 | alpha-crystallinopathy
http://purl.obolibrary.org/obo/MONDO_0016599 | autosomal dominant secondary polycythemia
http://purl.obolibrary.org/obo/MONDO_0016109 | autosomal recessive distal myopathy
http://purl.obolibrary.org/obo/MONDO_0008406 | autosomal recessive Emery-Dreifuss muscular dystrophy
http://purl.obolibrary.org/obo/MONDO_0016647 | autosomal recessive Stickler syndrome
http://purl.obolibrary.org/obo/MONDO_0018384 | avascular necrosis of genetic origin
http://purl.obolibrary.org/obo/MONDO_0016034 | cleft lip with or without cleft palate
http://purl.obolibrary.org/obo/MONDO_0017448 | congenital absence/hypoplasia of fingers excluding thumb
http://purl.obolibrary.org/obo/MONDO_0017447 | congenital absence/hypoplasia of thumb
http://purl.obolibrary.org/obo/MONDO_0018292 | congenital disorder of glycosylation-related bone disorder
http://purl.obolibrary.org/obo/MONDO_0018277 | congenital muscular dystrophy with cerebellar involvement
http://purl.obolibrary.org/obo/MONDO_0018279 | congenital muscular dystrophy without intellectual disability
http://purl.obolibrary.org/obo/MONDO_0015765 | congenital myopathy with cores
http://purl.obolibrary.org/obo/MONDO_0016121 | congenital myotonia
http://purl.obolibrary.org/obo/MONDO_0018701 | congenital nemaline myopathy
http://purl.obolibrary.org/obo/MONDO_0016524 | congenital vascular bone syndrome
http://purl.obolibrary.org/obo/MONDO_0016899 | Duchenne and Becker muscular dystrophy
http://purl.obolibrary.org/obo/MONDO_0018454 | dysostosis of genetic origin
http://purl.obolibrary.org/obo/MONDO_0800094 | dysostosis with brachydactyly with extraskeletal manifestations
http://purl.obolibrary.org/obo/MONDO_0800093 | dysostosis with brachydactyly without extraskeletal manifestations
http://purl.obolibrary.org/obo/MONDO_0800085 | dysostosis with predominant craniofacial involvement
http://purl.obolibrary.org/obo/MONDO_0800075 | dysostosis with predominant vertebral with and without costal involvement
http://purl.obolibrary.org/obo/MONDO_0800090 | ectrodactyly with and without other manifestations
http://purl.obolibrary.org/obo/MONDO_0015470 | familial isolated dilated cardiomyopathy
http://purl.obolibrary.org/obo/MONDO_0019150 | familial isolated restrictive cardiomyopathy
http://purl.obolibrary.org/obo/MONDO_0015668 | hereditary dentin defect
http://purl.obolibrary.org/obo/MONDO_0800092 | hereditary inflammatory or rheumatoid-like osteoarthropathy
http://purl.obolibrary.org/obo/MONDO_0018779 | hypercontractile muscle stiffness syndrome
http://purl.obolibrary.org/obo/MONDO_0015497 | hypoglossia/aglossia
http://purl.obolibrary.org/obo/MONDO_0016125 | infectious, fungal or parasitic myopathy
http://purl.obolibrary.org/obo/MONDO_0017420 | intercalary limb defects
http://purl.obolibrary.org/obo/MONDO_0018114 | isolated brachycephaly
http://purl.obolibrary.org/obo/MONDO_0015337 | isolated craniosynostosis
http://purl.obolibrary.org/obo/MONDO_0016520 | isolated Klippel-Feil syndrome
http://purl.obolibrary.org/obo/MONDO_0018113 | isolated plagiocephaly
http://purl.obolibrary.org/obo/MONDO_0018112 | isolated scaphocephaly
http://purl.obolibrary.org/obo/MONDO_0017429 | joint formation defects
http://purl.obolibrary.org/obo/MONDO_0015418 | lateral facial cleft
http://purl.obolibrary.org/obo/MONDO_0019718 | lethal chondrodysplasia
http://purl.obolibrary.org/obo/MONDO_0015412 | median facial cleft
http://purl.obolibrary.org/obo/MONDO_0019697 | mesomelic and rhizo-mesomelic dysplasia
http://purl.obolibrary.org/obo/MONDO_0017950 | microcephalic primordial dwarfism
http://purl.obolibrary.org/obo/MONDO_0019692 | multiple epiphyseal dysplasia and pseudoachondroplasia
http://purl.obolibrary.org/obo/MONDO_0019693 | multiple metaphyseal dysplasia
http://purl.obolibrary.org/obo/MONDO_0034671 | non-syndromic complex polydactyly
http://purl.obolibrary.org/obo/MONDO_0019713 | non-syndromic limb reduction defect
http://purl.obolibrary.org/obo/MONDO_0019714 | non-syndromic polydactyly, syndactyly and/or hyperphalangy
http://purl.obolibrary.org/obo/MONDO_0034670 | non-syndromic postaxial polydactyly
http://purl.obolibrary.org/obo/MONDO_0034669 | non-syndromic preaxial polydactyly
http://purl.obolibrary.org/obo/MONDO_0017421 | non-syndromic terminal limb defects
http://purl.obolibrary.org/obo/MONDO_0015415 | oblique facial cleft
http://purl.obolibrary.org/obo/MONDO_0018385 | osteochondrosis of genetic origin
http://purl.obolibrary.org/obo/MONDO_0015482 | otomandibular dysplasia
http://purl.obolibrary.org/obo/MONDO_0800091 | overgrowth or tall stature syndrome with skeletal involvement
http://purl.obolibrary.org/obo/MONDO_0015499 | paralytic facial malformation
http://purl.obolibrary.org/obo/MONDO_0015478 | paramedian facial cleft
http://purl.obolibrary.org/obo/MONDO_0019712 | patellar dysostosis
http://purl.obolibrary.org/obo/MONDO_0019689 | perlecan-related bone disorder
http://purl.obolibrary.org/obo/MONDO_0800089 | primary bone dysplasia with disorganized development of skeletal components
http://purl.obolibrary.org/obo/MONDO_0800084 | primary bone dysplasia with increased bone density
http://purl.obolibrary.org/obo/MONDO_0800086 | primary bone dysplasia with multiple joint dislocations
http://purl.obolibrary.org/obo/MONDO_0019699 | slender bone dysplasia
http://purl.obolibrary.org/obo/MONDO_0017423 | split hand or/and split foot malformation
http://purl.obolibrary.org/obo/MONDO_0019688 | sulfation-related bone disorder
http://purl.obolibrary.org/obo/MONDO_0800095 | syndrome with synostosis or other joint formation defect
http://purl.obolibrary.org/obo/MONDO_0015338 | syndromic craniosynostosis
http://purl.obolibrary.org/obo/MONDO_0016221 | temporomandibular joint anomaly
http://purl.obolibrary.org/obo/MONDO_0019176 | trichorhinophalangeal syndrome type I or III
http://purl.obolibrary.org/obo/MONDO_0018240 | TRPV4-related bone disorder
http://purl.obolibrary.org/obo/MONDO_0800087 | type 11 collagen-related bone disorder

**Comment by sabrinatoro** [comment]
Note: MONDO:0018240 was removed from the list of obsoletion candidate

**Comment by nicolevasilevsky** [comment]
[Spreadsheet here](https://docs.google.com/spreadsheets/d/1J5ell8LiRPgWLRJf13HVgEEL8eGV26O6De1897oIxjE/edit#gid=782346207)

**Comment by sabrinatoro** [comment]
Comment from Sylvia's comment on the spreadsheet:
According to PMID:28623672 craniosynostosis may be an isolated condition or part of complex syndromes. Is the elimination of  MONDO:0015337 isolated craniosynostosis really sensible?
 Is it really sensible to eliminate MONDO:0015338 syndromic craniosynostosis?

Also, discussions on the curation call (231026) suggested representing diseases as "isolated" and "syndromic" (part of a multi-system syndrome). 

Based on the opinion of the experts, I will remove MONDO:0015337 isolated craniosynostosis and MONDO:0015338 syndromic craniosynostosis from the obsoletion candidate list.

**Comment by sabrinatoro** [comment]
Also removed MONDO:0019054 (congenital limb malformation) from the obsoletion list since it is used in ClinGen

**Comment by twhetzel** [comment]
fwiw, i highlighted the children of MONDO:0015337 isolated craniosynostosis and MONDO:0015338 syndromic craniosynostosis in gray in the spreadsheet to indicate _something_ different happened with these rows. not the best, but a small step better than no indication in the spreadsheet that anything different happened to these rows.

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1yJ08nxP3Eu59G3wwVO8KXc6gw6Mh21D2/edit#gid=547601471 


TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab) !!!!1 term comment will need to be added manually
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)
- [ ] review "Reviewer-Leave Curator-Stay" Tab [TO DO WILL BE UPDATED ACCORDINGLY]


Note: Once the to-do list is done, 213 terms for which reviewers were unsure will remain to be assessed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
